Scripts/BuildUtils.ps1

```diff
@@ -0,0 +1,71 @@
+$DefaultAccountName = "mirasrael"
+$ProjectSlug = "runtimegraphicsinfo"
+$RuntimesPath = "$PSScriptRoot\..\Runtime\Plugins"
+
+function BuildUtils-DownloadAppVeyorRuntimes([string]$Token = "", [string]$AccountName = "", [string]$BuildId = "") {
+    $apiUrl = 'https://ci.appveyor.com/api'
+
+    if ($Token -eq "")
+    {
+        $tokenFilePath = "$HOME/.appveyor/token"
+        if ([System.IO.File]::Exists($tokenFilePath))
```

Comment:
in powershell you can just use `Test-Path $tokenFilePath`

Reply:
changed